File: app/src/main/java/com/fernandez/pablo/la24gnc/Presenter/MainPresenter.java
package com.fernandez.pablo.la24gnc.Presenter;

import android.content.Context;

import com.fernandez.pablo.la24gnc.Model.TurnoDAO;

/**
 * Created by pablo on 10/05/2017.
 */

public class MainPresenter {

    public static String hayTurnoAbierto(Context context){

        return TurnoDAO.hayTurnoAbierto(context);

    }

}

File: app/src/main/java/com/fernandez/pablo/la24gnc/Presenter/MainPresenter.kt
package com.fernandez.pablo.la24gnc.Presenter

import android.content.Context

import com.fernandez.pablo.la24gnc.Model.TurnoDAO

/**
 * Created by pablo on 10/05/2017.
 */

object MainPresenter {

    fun hayTurnoAbierto(context: Context): String? {

        return TurnoDAO.hayTurnoAbierto(context)

    }

}
